modules/store.js

```diff
@@ -0,0 +1,33 @@
+// Store Class: Handles Storage
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
```

Comment:
- [ ] [OPTIONAL] I'd highly advise you to use array methods that don't mutate the data directly as this could lead to data inconsistency when your project grows large.
- [ ] You could use `books = books.concat(book);`